script/Arcueid/Day 7/07_00_ARC07_3E.txt

```diff
@@ -218,7 +216,7 @@ Her gaze is trained on a run-of-the-mill looking man in a business suit as he wa
 [sha:4bcc9ad36016750ac0b7cfadd5b87327e6dcbbed]{
 -- Page 289, Offset 17280. Mods: @n.
 -- 「早く。
-"Hurry up.
+"Hurry.
```

Comment:
Mild pref for original, shortening it to _just_ hurry makes it feel a bit more dire than it is